po/es.po

```diff
@@ -366,43 +366,43 @@ msgstr "Ejemplo"
 
 #: src/SUMMARY.md:111 src/exercises/day-2/health-statistics.md:1
 msgid "Health Statistics"
-msgstr "Estadísticas de Estado de Salud"
+msgstr "Estadísticas de salud"
 
 #: src/SUMMARY.md:113
 msgid "Day 2: Afternoon"
-msgstr "Día 2: Tarde"
+msgstr "Día 2: tarde"
 
 #: src/SUMMARY.md:115 src/std.md:1
 msgid "Standard Library"
-msgstr "Librería Standard"
+msgstr "Biblioteca estándar"
 
 #: src/SUMMARY.md:116
 msgid "Option and Result"
 msgstr "Option y Result"
 
 #: src/SUMMARY.md:117 src/std/string.md:1
 msgid "String"
-msgstr ""
+msgstr "String"
 
 #: src/SUMMARY.md:118
 msgid "Vec"
-msgstr ""
+msgstr "Vec"
 
 #: src/SUMMARY.md:119
 msgid "HashMap"
-msgstr ""
+msgstr "HashMap"
 
 #: src/SUMMARY.md:120
 msgid "Box"
-msgstr ""
+msgstr "Box"
 
 #: src/SUMMARY.md:121
 msgid "Recursive Data Types"
-msgstr "Tipo de Datos Recursivos"
+msgstr "Tipos de datos recursivos"
 
 #: src/SUMMARY.md:122 src/std/box-niche.md:1
 msgid "Niche Optimization"
-msgstr "Optimización de Nicho"
+msgstr "Optimización de la Memória"
```

Comment:
It's "memoria" not "memória" - drop the accent. https://dle.rae.es/memoria

Also in spanish we don't capitalize words of a title, make it lowercase "memoria", not "Memoria".

Reply:
Hey @deavid, you can speed up the review process a lot by using the GitHub suggestion feature (the left-most button in the toolbar above):

```suggestion
msgstr "Optimización de la memoria"
```

That way the @henrif75 can fix the translation with a single click afterwards.

Reply:
Thanks for pointing that out. I'll try it on my next review. I have a lot to do.

Reply:
I didn't know about the feature myself until recently! It has saved a lot of review round-trips :smile: